Source/Csla.Blazor.WebAssembly.Tests/Csla.Blazor.WebAssembly.Tests.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net8.0</TargetFrameworks>
+    <IsPackable>false</IsPackable>
+    <ApplicationIcon />
+    <OutputType>Library</OutputType>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <PackageReference Include="Microsoft.Extensions.Configuration" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Binder" Version="8.0.0" />
+    <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="8.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.9.0" />
+    <PackageReference Include="Moq" Version="4.20.70" />
```

Comment:
@rockfordlhotka We/You have to decide if we want to depent on Moq. I personally don't like Moq anymore due to the "sponsor link" incident last year. (c.f. https://github.com/devlooped/moq/issues/1372 and a lot more issues/discussion in th repo)

Reply:
@StefanOssendorf  I read about this issue and the latest release https://github.com/devlooped/moq/releases/tag/v4.20.70 removed this with this PR https://github.com/devlooped/moq/pull/1402

Reply:
That's true. But it's still a loss of trust.

Reply:
What can we do to finalize this?

Reply:
Changes to NSubstitute